powershell/admin-center/Microsoft.PowerApps.Administration.PowerShell.Samples.psm1

```diff
@@ -1837,6 +1838,118 @@ function CleanTestPolicies
     }
 }
 
+function IncludeInsightsForManagedEnvironmentsInPPACHomePageCards
```

Comment:
I wonder if it possible to check if tenant level analytics is enabled. first. Same comment for weekly digest.